### PR TITLE
HTML structure committ

### PR DESCRIPTION
**Pull request by carolllemieux**
structure added to webpages/ header main section and footer.

### CODE REVIEW

**Review by axamei — CHANGES_REQUESTED** [review]
i don't see a folder with images. is this something missing intentionally?

**Comment by carolllemieux** [comment]
I have an image folder. I think I made it incorrectly. I went to the github folder in my documents folder and created the image folder. I didn't realize I should make it a different way. I don't quite understand how to fix it so that Github knows the folder is there.

**Review by carolllemieux — COMMENTED** [review]
I am confused. The things highlighted in red I removed or changed during the HTML assignment process. I don't understand why they are in the document you are seeing. How do I make the changes and resubmit? 

Do I need to merge, then make the changes or do I make the changes and then create a new committ?

**Review by carolllemieux — COMMENTED** [review]
The index page here is not the one that is on my Visual Code. I don't understand why there is a discrepancy between them. My GitHub doesn't says I have 0 changes.

File: contact-me.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Me</title>
+</head>
+<body>
+    <section>
+        <header>
+            <section>
+                <figure> 
+                    <img src="../ss22-carolllemieux/images/logocl.PNG" alt="Logo overlapping CL Carol LeMieux's initials" style="width:200px;height:200px;">
```
Comment (axamei) [inline contact-me.html]: Change `src="../ss22-carolllemieux/images/logocl.PNG"` to `src="images/logocl.PNG"`

File: contact-me.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                
+                    <img src="../ss22-carolllemieux/images/office.jpg" alt="technology office" style="width:600px;height:200px;"></figure>
```
Comment (axamei) [inline contact-me.html]: Change the `src` to remove `../ss22-carolllemieux/`

File: edtech.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Educational Technology</title>
+</head>
+<body>
+    <section>
+        <header>
+            <section>
+                <figure> 
+                    <img src="../ss22-carolllemieux/images/logocl.PNG" alt="Logo overlapping CL Carol LeMieux's initials" style="width:200px;height:200px;">
```
Comment (axamei) [inline edtech.html]: Change the `src` to remove `../ss22-carolllemieux/`

File: edtech.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                
+                    <img src="../ss22-carolllemieux/images/office.jpg" alt="technology office" style="width:600px;height:200px;"></figure>
```
Comment (axamei) [inline edtech.html]: Change the `src` to remove `../ss22-carolllemieux/`

File: index.html [hunk 5]
```diff
@@ -3,13 +3,87 @@
...
-    <p>Carol LeMieux</p> 
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/VSIQO3a8IPs" 
-    title="YouTube video player" frameborder="0" 
-    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; 
-     picture-in-picture; web-share" allowfullscreen></iframe> 
+    <section>
+        <header>
+            <section>
+                <figure> 
+                    <img src="../ss22-carolllemieux/images/logocl.PNG" alt="Logo overlapping CL Carol LeMieux's initials" style="width:200px;height:200px;">
```
Comment (axamei) [inline index.html]: Change the `src` to remove `../ss22-carolllemieux/`

File: index.html [hunk 6]
```diff
@@ -3,13 +3,87 @@
...
+                
+                    <img src="../ss22-carolllemieux/images/office.jpg" alt="technology office" style="width:75%;height:200px;"></figure>
```
Comment (axamei) [inline index.html]: Change the `src` to remove `../ss22-carolllemieux/`

File: mlis.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MLIS</title>
+</head>
+<body>
+    <section>
+        <header>
+            <section>
+                <figure> 
+                    <img src="../ss22-carolllemieux/images/logocl.PNG" alt="Logo overlapping CL Carol LeMieux's initials" style="width:200px;height:200px;">
```
Comment (axamei) [inline mlis.html]: Change the `src` to remove `../ss22-carolllemieux/`

File: mlis.html [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+                
+                    <img src="../ss22-carolllemieux/images/office.jpg" alt="technology office" style="width:600px;height:200px;"></figure>
```
Comment (axamei) [inline mlis.html]: Change the `src` to remove `../ss22-carolllemieux/`

File: resume.html [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Resumé</title>
+</head>
+<body>
+    <section>
+        <header>
+            <section>
+                <figure> 
+                <img src="../ss22-carolllemieux/images/logocl.PNG" alt="Logo overlapping CL Carol LeMieux's initials" style="width:200px;height:200px;">
```
Comment (axamei) [inline resume.html]: Change the `src` to remove `../ss22-carolllemieux/`

File: resume.html [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+            
+                <img src="../ss22-carolllemieux/images/office.jpg" alt="technology office" style="width:600px;height:200px;"></figure>
```
Comment (axamei) [inline resume.html]: Change the `src` to remove `../ss22-carolllemieux/`

File: testimonials.html [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Testimonials</title>
+</head>
+<body>
+    <section>
+        <header>
+            <section>
+                <figure> 
+                    <img src="../ss22-carolllemieux/images/logocl.PNG" alt="Logo overlapping CL Carol LeMieux's initials" style="width:200px;height:200px;">
```
Comment (axamei) [inline testimonials.html]: Change the `src` to remove `../ss22-carolllemieux/`

File: testimonials.html [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+                
+                    <img src="../ss22-carolllemieux/images/office.jpg" alt="technology office" style="width:600px;height:200px;"></figure>
```
Comment (axamei) [inline testimonials.html]: Change the `src` to remove `../ss22-carolllemieux/`

**Review by axamei — CHANGES_REQUESTED** [review]
Change the `src` to remove `../ss22-carolllemieux/`. You can use the find/replace tool in VS Code or change the individual HTML files.

**Comment by carolllemieux** [comment]
I have corrected the img src code in each file.

**Review by axamei — APPROVED** [review]

**Pull request merged**